spec/requests/schools/dashboard_spec.rb

```diff
@@ -4,12 +4,20 @@
 
 RSpec.describe "Schools::Dashboard", type: :request do
   before do
-    user = create(:user, :induction_coordinator)
-    sign_in user
+    @user = create(:user, :induction_coordinator)
```

Comment:
If you need to refer to user then use a let(:user) rather than @user

Reply:
Isn't that for when it's persisted across tests? In this case, it shouldn't be

(I also have no idea what I'm talking about, feel free to correct me here!)

Reply:
No, that's when you want variables that are lazily instantiated. Exactly, what is needed here.